rpcs3/Emu/Cell/PPULLVMRecompiler.cpp

```diff
@@ -7,7 +7,6 @@
 #include "llvm/Support/Host.h"
 #include "llvm/Support/ManagedStatic.h"
 #include "llvm/ExecutionEngine/GenericValue.h"
-#include "llvm/ExecutionEngine/MCJIT.h"

```

Comment:
Well, you could delete MCJIT in Core as well https://github.com/Zangetsu38/rpcs3/blob/76a0113b4dfc2b68f035707c358193f6508bf6ef/rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp#L11